src/engine/BMGameAction.php

```diff
@@ -501,8 +501,13 @@ protected function messageAttacker($preAttackDice, $postAttackDice, $attackType
                 $postEventsAttacker,
                 $this->message_recipe_change($attackerInfo, $postInfo)
             );
-            if (!array_key_exists('forceHideDieReroll', $postInfo) ||
-                ($attackType == 'Trip')) {
+            if (array_key_exists('forceHideDieReroll', $postInfo) &&
+                !($attackType == 'Trip')) {
+                $this->message_append(
+                    $postEventsAttacker,
+                    'rerolls from ' . $attackerInfo['value'] . ', changes size due to turbo'
```

Comment:
Nit: i think everything else in the log is in past tense ("rerolled", "changed"), so this should be too.